news/2020-06-15-cwc-2020-concludes.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "osu!catch World Cup 2020 Concludes"
+date: 2020-06-15 14:00:00 +0000
+---
+
+At the conclusion of six weeks of fast and furious fruit-catching action, who rose to the challenge and claimed this year's title of *osu!catch Champions*? Read inside to find out all the juicy details!
+
+![](/wiki/shared/news/banners/CWC2020.jpg)
+
+The osu!catch World Cup has a rich history of titanic rivalries, clutch come comebacks and underdog Cinderella stories, and this year's tournament was no exception. Whether it be Saudia Arabia looking right at home in their first ever World Cup appearance, Italy playing three weeks' of consecutive TieBreakers, or even the eternal Twitch chat debate of whether [frogs](https://osu.ppy.sh/beatmapsets/12221) are cooler than [cats](https://osu.ppy.sh/beatmapsets/1119839), this year once again gave us some truly memorable moments!
+
+However, it would be China, South Korea and The United States who would rise above the rest and do battle for the final time in the Grand Finals. Feel free to read on for rapid rundopwn, but if you find yourself with time to spare during the lockdown, you can relive the greatest matches and moments on the [osu!live Twitch channel](https://www.twitch.tv/osulive), along with all previous rounds if you need a refresher!
```

Comment:
```suggestion
However, it would be China, South Korea and The United States who would rise above the rest and do battle for the final time in the Grand Finals. Feel free to read on for a rapid rundown, but if you find yourself with time to spare during the lockdown, you can relive the greatest matches and moments on the [osu!live Twitch channel](https://www.twitch.tv/osulive), along with all previous rounds if you need a refresher!
```